src/components/GoalCreateForm/GoalCreateForm.tsx

```diff
@@ -32,38 +32,36 @@ const StyledButtonWithDropdown = styled.div`
     align-items: center;
 `;
 
+const createOptions = [
```

Comment:
It's wrong and doesn't work because there are no context about locale. You can check it in ru local for example.